### DISCUSSION TITLE
Does RBM have the ability to automatically benchmark and compare two different overclock profiles for the same miner/algo/pool?

### DISCUSSION THREAD

**Discussion by coolbreeze21** [RainbowMiner/RainbowMiner#1914] [Q&A] [answered]
It would be very useful if RBM had the ability to automatically compare two (or more) overclock profiles for the same miner/algo/pool.  This would be similar to how it benchmarks different miners on the same algo and identifies the most profitable miner.  I noticed that the “Miners” tab in the browser interface has “Speed 1” and “Speed 2” columns, but only “Speed 1” has data in it on my systems.  That led me to think that RBM might already have the ability to do this, but I searched the Issues and Discussions pages on the RBM GitHub and didn’t see any mention of it.  If this capability already exists in RBM, how do you use it?

I would guess that anyone who has ever overclocked for mining has done something similar with pen, paper and/or spreadsheet, but RBM’s robust data collection and built-in OC profile capabilities would make the process so much easier.  It could quickly answer questions like “how much GPU core underclock is most profitable for ETH on my card?”.

**Comment by RainbowMiner** [comment, marked as answer]
RainbowMiner currently doesn't run multiple overclocking profiles for one miner/algo/pool. The "Speed 2" column is for miners, that support dual mining. 
Anyway, I admit that such a feature would be nice. I'll re-think this idea.

**Reply by coolbreeze21** [reply]
Thanks!  If RBM has the ability to switch Afterburner and/or Nvidia profiles while a miner is running, it could make for relatively quick benchmarking - no relaunching of miners, reconnecting with pools, rebuilding dags.  Just get the miner/pool/algo running and cycle through the selected OC profiles for the benchmarking interval and collect the data.
